### PR TITLE
XEP-0162: Fix small typos

### PR DESCRIPTION
**Pull request by vanitasvitae**

### CODE REVIEW

File: xep-0162.xml [hunk 1]
```diff
@@ -23,6 +23,10 @@
     <email>lucas@lucas-nussbaum.net</email>
     <jid>lucas@nussbaum.fr</jid>
   </author>
+  <revision>
```
Comment (vanitasvitae) [inline xep-0162.xml:26]: Still needs date + version number

**Comment by horazont** [comment]
@vanitasvitae There is a validity error in there, please see the build logs.

**Comment by vanitasvitae** [comment]
Yeah, this is because version and date are missing. Should I simply set version to 0.2.1 and date to today?

**Comment by horazont** [comment]
@vanitasvitae Either no revision block (in that case preferrably with a description or PR title the editors can reuse when creating one), or a complete one. Please no in-between thing.

**Comment by vanitasvitae** [comment]
Fixed :)

**Pull request merged**